server/utils/utils.ts

```diff
@@ -192,6 +197,25 @@ const selectReleaseDate = (nomisRecord: Prisoner) => {
   return dateString
 }
 
+const selectReleaseDateFromLicence = (licence: Licence) => {
```

Comment:
util libraries should generally be completely agnostic to the domain you are working on. 

These are really business/domain rules that should sit with `services`, granted they could live in a module of functions inside there.